pwem/tests/protocols/test_protocol_mathematical_operator.py

```diff
@@ -0,0 +1,110 @@
+# ***************************************************************************
+# * Authors:    Daniel Marchán (da.marchan@cnb.csic.es)
+# *
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 3 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# ***************************************************************************/
+
+
+
+from pyworkflow.tests import BaseTest, setupTestProject, DataSet
+from pyworkflow.plugin import Domain
+from pwem.protocols.protocol_import import ProtImportMicrographs
+from pwem.protocols.protocol_mathematical_operator import ProtMathematicalOperator
+
+SphireProtCRYOLOPicking = Domain.importFromPlugin('sphire.protocols', 'SphireProtCRYOLOPicking', doRaise=True)
```

Comment:
We should avoid other plugins dependencies if possible.

In this case, it is easy to make the test using other protocol from this plugin instead of the one from sphire. I suggest to adapt this.